app/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,221 @@
+<resources>
+    <!-- Outras Strings -->
```

Comment:
Please don't translate these comments; they're there to help me and other contributors identify which strings belong where. So when I need to extend an existing feature, I can quickly find and update the strings related to that. If you want to add translations anyway, then do it like this:

`<!-- original title | translated title -->`

Reply:
Done.

Reply:
It's not exactly hardcoded strings, I should have explained better.

You are using the `Enum.name` in several places this practice breaks the localization and the end result is the same as a hardcoded string.

Example:
GoalPriority.High.name